### DISCUSSION TITLE
youtube template variable for video duration

### DISCUSSION THREAD

**Discussion by bmaniscalco** [DominikPieper/obsidian-ReadItLater#77] [Ideas]
It would be really handy if the youtube template provided a variable for video duration. Then e.g. video length could be used in dataview queries to sort through videos in the vault depending on how much time is currently available for watching.

**Comment by adamluckdev** [comment]
Similar topic, answer from other thread applies also for this one.

https://github.com/DominikPieper/obsidian-ReadItLater/discussions/73#discussioncomment-4823253

**Comment by adamluckdev** [comment]
Released in [0.2.0](https://github.com/DominikPieper/obsidian-ReadItLater/releases/tag/0.2.0)

**Reply by gwillcox-r7** [reply]
Whilst its documented in README.md, it seems that opening the plugin and going to the settings and then `Youtube Note Template` doesn't show this in the list of available template items. Might need to update the list again? Looks like this list is being maintained in two locations: README.md and https://github.com/DominikPieper/obsidian-ReadItLater/blob/ce275288ea429e9f7d99b75b537d19caf6093bf7/src/views/settings-tab.ts at the moment.
